src/System.Linq/tests/OrderedSubsetting.cs

```diff
@@ -223,7 +223,12 @@ public void TakeAndSkip()
             Assert.Empty(ordered.Skip(100).Take(20));
             Assert.Equal(Enumerable.Range(10, 20), ordered.Take(30).Skip(10));
             Assert.Equal(Enumerable.Range(10, 1), ordered.Take(11).Skip(10));
+        }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "This fails with an OOM with the full .NET framework, as it iterates through the large array. See https://github.com/dotnet/corefx/pull/6821.")]
```

Comment:
I'm using this PR as this is where the test case was added: https://github.com/dotnet/corefx/pull/6821/files#diff-3d4b03f25eb943838fdf80384ffca61bR227

@JonHanna FYI